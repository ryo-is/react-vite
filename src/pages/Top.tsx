export const Top = () => {
  const title = 'hello!';

  return <div>{title}</div>;
};
