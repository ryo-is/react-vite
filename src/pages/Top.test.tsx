import { render, screen } from '@testing-library/react';
import { Top } from './Top';

describe('Top', () => {
  it('should match to the snapshot', () => {
    const { asFragment } = render(<Top />);
    expect(asFragment()).toMatchSnapshot();
  });

  it('should render "hello!"', () => {
    render(<Top />);
    screen.debug();
    expect(screen.getByText('hello!')).toBeInTheDocument();
  });
});
