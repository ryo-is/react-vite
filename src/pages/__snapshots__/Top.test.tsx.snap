// Vitest Snapshot v1, https://vitest.dev/guide/snapshot.html

exports[`Top > should match to the snapshot 1`] = `
<DocumentFragment>
  <div>
    hello!
  </div>
</DocumentFragment>
`;
